sdk/confidentialledger/confidential-ledger-rest/test/snippets.spec.ts

```diff
@@ -73,6 +73,31 @@ describe("snippets", () => {
     const result = await client.path("/app/transactions").post(ledgerEntry);
   });
 
+  it("ReadmeSamplePostLedgerEntryWithCollectionIdAndTags", async () => {
+    const { ledgerIdentityCertificate } = await getLedgerIdentity(
+      "test-ledger-name",
+      "https://identity.confidential-ledger.core.azure.com",
+    );
+    const credential = new DefaultAzureCredential();
+    // @ts-preserve-whitespace
+    const client = ConfidentialLedger(
+      "https://test-ledger-name.confidential-ledger.azure.com",
+      ledgerIdentityCertificate,
+      credential,
+    );
+    // @ts-preserve-whitespace
+    const entry: LedgerEntry = {
```

Comment:
I wonder whether you are using an IDE (VS CODE) to edit the snippets.spec.ts file?  I am seeing red squiggles indicating that `collectionId` is not a valid property of `LedgerEntry`, Neither is `tags`

![image](https://github.com/user-attachments/assets/5370d292-afde-427d-b309-64a92d2a8105)